modules/tinymce/src/core/test/ts/browser/init/ShadowDomEditorTest.ts

```diff
@@ -26,11 +26,7 @@ UnitTest.asynctest('Skin stylesheets should be loaded in ShadowRoot when editor
   TinyLoader.setupFromElement((editor: Editor, onSuccess, onFailure) => {
     Pipeline.async({}, [
       Step.sync(() => {
-        Assert.eq(
-          'There should not be any skin stylesheets in the document',
-          false,
-          Arr.exists(document.styleSheets, isSkin)
-        );
+        // TODO: Test that there are no skin stylesheets in the head. We will need to clean up existing stylesheets first, which may require a StyleSheetLoader.removeAll() function
```

Comment:
Should you add the JIRA-issue for doing this in the comment?

Reply:
Good idea. Done.